itur_bt500_demo/README.md

```diff
@@ -0,0 +1,329 @@
+5.0,3.0,5.0,5.0,5.0,1.0,4.0,2.0,1.0,3.0,1.0,2.0,2.0,1.0,2.0,1.0,2.0,1.0,1.0,2.0,2.0,1.0,2.0,2.0,3.0,2.0
+3.0,3.0,4.0,4.0,1.0,2.0,3.0,3.0,1.0,4.0,3.0,3.0,2.0,2.0,3.0,2.0,2.0,2.0,2.0,1.0,3.0,2.0,2.0,2.0,3.0,2.0
+1.0,5.0,4.0,2.0,5.0,4.0,5.0,4.0,3.0,5.0,2.0,3.0,4.0,3.0,2.0,2.0,2.0,2.0,3.0,3.0,3.0,3.0,3.0,2.0,4.0,3.0
+2.0,1.0,3.0,4.0,5.0,4.0,5.0,5.0,3.0,5.0,4.0,5.0,5.0,3.0,5.0,5.0,4.0,4.0,3.0,4.0,4.0,4.0,4.0,4.0,5.0,4.0
+5.0,3.0,4.0,5.0,3.0,5.0,5.0,5.0,4.0,5.0,4.0,5.0,5.0,4.0,3.0,5.0,4.0,4.0,4.0,4.0,5.0,4.0,4.0,5.0,5.0,3.0
+4.0,2.0,2.0,1.0,5.0,5.0,4.0,5.0,5.0,5.0,4.0,5.0,5.0,4.0,5.0,5.0,5.0,4.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0
+5.0,5.0,4.0,1.0,5.0,1.0,1.0,1.0,1.0,2.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0
+2.0,4.0,5.0,2.0,5.0,2.0,2.0,3.0,1.0,2.0,2.0,3.0,4.0,3.0,2.0,2.0,2.0,1.0,1.0,1.0,2.0,2.0,2.0,1.0,2.0,2.0
+5.0,2.0,4.0,5.0,3.0,1.0,3.0,4.0,2.0,4.0,3.0,4.0,3.0,3.0,3.0,3.0,2.0,2.0,2.0,3.0,2.0,2.0,3.0,1.0,3.0,3.0
+4.0,1.0,4.0,5.0,2.0,5.0,3.0,4.0,3.0,5.0,3.0,3.0,3.0,4.0,3.0,3.0,4.0,2.0,2.0,4.0,3.0,3.0,2.0,2.0,3.0,4.0
+2.0,1.0,5.0,5.0,2.0,3.0,3.0,3.0,3.0,5.0,4.0,4.0,5.0,3.0,5.0,3.0,3.0,3.0,3.0,4.0,4.0,3.0,4.0,3.0,5.0,4.0
+3.0,5.0,5.0,4.0,5.0,2.0,3.0,nan,4.0,5.0,3.0,4.0,4.0,4.0,4.0,3.0,4.0,3.0,2.0,4.0,4.0,3.0,4.0,3.0,5.0,4.0
+3.0,5.0,4.0,4.0,4.0,5.0,4.0,5.0,5.0,5.0,3.0,5.0,4.0,4.0,5.0,4.0,5.0,3.0,4.0,4.0,3.0,5.0,3.0,4.0,4.0,5.0
+3.0,3.0,3.0,5.0,4.0,5.0,4.0,4.0,4.0,5.0,4.0,5.0,5.0,5.0,5.0,5.0,5.0,3.0,4.0,5.0,5.0,5.0,4.0,4.0,5.0,5.0
+3.0,3.0,5.0,3.0,5.0,1.0,3.0,5.0,3.0,5.0,4.0,4.0,5.0,4.0,5.0,4.0,5.0,4.0,3.0,5.0,4.0,4.0,4.0,5.0,5.0,5.0
+2.0,2.0,3.0,5.0,4.0,5.0,4.0,4.0,4.0,5.0,4.0,5.0,4.0,3.0,4.0,5.0,5.0,4.0,4.0,5.0,5.0,5.0,4.0,5.0,5.0,5.0
+1.0,1.0,1.0,5.0,3.0,1.0,1.0,3.0,1.0,3.0,1.0,1.0,2.0,1.0,1.0,1.0,2.0,2.0,2.0,2.0,1.0,2.0,1.0,1.0,3.0,1.0
+3.0,4.0,2.0,5.0,5.0,2.0,5.0,3.0,3.0,4.0,3.0,3.0,4.0,3.0,3.0,2.0,2.0,2.0,2.0,3.0,2.0,2.0,2.0,2.0,2.0,3.0
+4.0,3.0,3.0,4.0,3.0,3.0,2.0,4.0,3.0,4.0,4.0,4.0,4.0,4.0,3.0,2.0,2.0,4.0,2.0,4.0,3.0,2.0,3.0,3.0,5.0,3.0
+2.0,5.0,5.0,3.0,4.0,4.0,3.0,4.0,1.0,4.0,3.0,4.0,5.0,3.0,3.0,4.0,3.0,3.0,3.0,3.0,2.0,3.0,3.0,4.0,4.0,3.0
+4.0,4.0,5.0,1.0,5.0,5.0,5.0,5.0,2.0,5.0,5.0,4.0,5.0,3.0,4.0,4.0,5.0,4.0,3.0,5.0,4.0,3.0,4.0,4.0,5.0,5.0
+3.0,4.0,3.0,2.0,5.0,5.0,4.0,5.0,4.0,5.0,5.0,4.0,5.0,3.0,5.0,4.0,5.0,5.0,5.0,5.0,5.0,4.0,5.0,4.0,5.0,4.0
+
+```
+If some subjects evaluated a PVS multiple times, another 2D matrix of the
+same size [num_PVS, num_subjects] can be added under the first one. A row
+of -1 should be placed before the repetition matrix. Where the repeated
+vote is not available, a 'nan' is put in place. See `small_sample_data.csv` file for an example:
+
+```
+...
+4.0,1.0,3.0,5.0,3.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0
+3.0,3.0,1.0,3.0,1.0,1.0,2.0,3.0,1.0,3.0,1.0,3.0,1.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0
+5.0,3.0,2.0,2.0,5.0,3.0,1.0,3.0,1.0,4.0,3.0,4.0,3.0,4.0,3.0,3.0,3.0,2.0,1.0,2.0
+-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0,-1.0
```

Comment:
I understand the motivation of adding an all-minus-1 row as a separator (to make the file still CSV-compatible), but it still feels a bit weird to me:)

I think the simplest and easiest to understood solution is just to use a single comma in the separator row. For example:
4.0,1.0,3.0,5.0,3.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0,1.0
3.0,3.0,1.0,3.0,1.0,1.0,2.0,3.0,1.0,3.0,1.0,3.0,1.0,2.0,2.0,2.0,2.0,2.0,2.0,2.0
5.0,3.0,2.0,2.0,5.0,3.0,1.0,3.0,1.0,4.0,3.0,4.0,3.0,4.0,3.0,3.0,3.0,2.0,1.0,2.0
,
5.0,nan,5.0,4.0,2.0,5.0,3.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0
1.0,3.0,5.0,2.0,5.0,5.0,5.0,5.0,4.0,5.0,4.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0,5.0
3.0,5.0,5.0,5.0,4.0,5.0,4.0,5.0,3.0,4.0,4.0,5.0,5.0,5.0,5.0,5.0,5.0,4.0,4.0,5.0

and this can be processed by 
csv.reader(datafile, delimiter=',')
correctly.